frontend/src/pages/admin/access/constants/constants.ts

```diff
@@ -0,0 +1,58 @@
+import { ColDef } from 'ag-grid-community';
+import { IAccessRequest } from 'interfaces';
+import { AccessStatusDisplay, AccessStatusDisplayMapper } from 'constants/accessStatus';
+
+export const columnDefinitions = (): ColDef[] => {
+  let items: ColDef[] = [
```

Comment:
does it make sense to shield params.data from NPE's here? I think the component itself is making sure not to display the grid if params.data isn't populated. Perhaps just documenting that params.data is required is good enough.

Reply:
You are right about that, no need to shield it